src/services/pay-services.ts

```diff
@@ -23,7 +23,7 @@ export default class PayServices {
     return axios.get(url)
       .then(response => {
         const fees = response?.data
-        if (!fees.filingFees) {
```

Comment:
This was throwing an error when fees.filingFees=0, ie,

![image](https://user-images.githubusercontent.com/10002889/210450519-e77fd0d2-abf8-4fdd-83c4-7c717c546360.png)